src/index.html

```diff
@@ -1,12 +1,329 @@
 <!doctype html>
-<html lang="en">
+<html lang="en" class="page">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
-  <body>
-    <h1>Catalog</h1>
+  <body class="page__body">
+    <header class="header">
+      <img
+        class="header__logo"
+        src="images/logo.png"
+        alt="Moyo logo"
+      >
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a class="nav__link is-active" href="#Apple">Apple</a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#Samsung">Samsung</a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#Smartphones">Smartphones</a>
+          </li>
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="#Laptops-&-Computers"
+              data-qa="nav-hover"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#Gadgets">Gadgets</a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#Tablets">Tablets</a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#Photo">Photo</a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#Video">Video</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main class="main">
+      <div class="card" data-qa="card">
+        <img
+          src="./images/imac.jpeg"
+          alt="image imac"
+          class="card__image"
+        >
+        <span class="card__title">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</span>
+        <span class="card__product-code">Product code: 195434</span>
+
+        <div class="card__rating">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+
+          <span class="card__rating-reviews">Reviews: 5</span>
+        </div>
+
+        <div class="card__price">
+          <span class="card__price-title">Price:</span>
+          <span class="card__price-amount">$2,199</span>
+        </div>
+
+        <a
+          href="#"
+          class="card__link"
+          data-qa="card-hover"
+        >
+          Buy
+        </a>
+      </div>
+      <div class="card">
```

Comment:
Add the blank line between the cards
```suggestion
      </div>
      
      <div class="card">
```